project_stock/models/stock_move.py

```diff
@@ -45,6 +46,7 @@ def _prepare_analytic_line_from_task(self):
             "product_uom_id": self.product_uom.id,
             "company_id": analytic_account.company_id.id or self.env.company.id,
             "partner_id": task.partner_id.id or task.project_id.partner_id.id or False,
+            "project_id": task.project_id.id,
```

Comment:
This was removed on purpose, as only timesheets have the project filled.

Reply:
Reviewing v14, I see that it auto-defined `project_id` with `hr_timesheet` (not sure how yet).

Reply:
But these are not timesheets. That's why you shouldn't set it.